ui/packages/statement/src/RootComponent.tsx

```diff
@@ -39,7 +40,7 @@ const App = withRouter((props) => {
           </Breadcrumb>
         </div>
         <div style={{ margin: 12 }}>
-          <Switch>
+          <Routes>
             <Route path="/statement/overview">
               <StatementsOverviewPage
                 dashboardClient={client.getInstance()}
```

Comment:
After my test, nest the component inside the `<Route>` doesn't work in v6, we need to make the component as `element` prop value, like this: `<Route path="/statement/overview" element={<StatementsOverviewPage .../>}`